tardis/montecarlo/src/cmontecarlo.c

```diff
@@ -829,41 +832,56 @@ montecarlo_main_loop(storage_model_t * storage, int64_t virtual_packet_flag, int
 #ifdef WITHOPENMP
   omp_set_dynamic(0);
   omp_set_num_threads(nthreads);
-#pragma omp parallel
+
+#pragma omp parallel firstprivate(finished_packets)
```

Comment:
why is this change in there? I don't think that has anything to do with the logging, right?